lib/house.rb

```diff
@@ -0,0 +1,47 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @above_market_price = "$500,000"
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def add_commas_to_price(price)
+    @price.to_s.reverse.gsub(/(\d{3})(?=\d)/, '\\1,').reverse
```

Comment:
i don't think there's anything in the interaction pattern that suggests that the price needs to have commas in it... so while I very much appreciate the effort you've shown here, it was not necessary, and the time probably could have been spent better elsewhere.

Reply:
Also, you never actually use this anywhere...